src/routes/index.js

```diff
@@ -41,4 +42,62 @@ const router = Router();
  */
 router.post('/users', createUser);
 
+/**
+ * @swagger
+ *
+ * /api/v1/users:
+ *   post:
+ *     description: Follow User
+ *     produces:
+ *       - application/json
+ *     request:
+ *         content:
+ *         - application/json
+ *         schema:
+ *           type: array
+ *           items:
+ *         $ref: '#/definitions/users'
+ *     responses:
+ *       201:
+ *         description: User followed
+ *       400:
+ *         description: Bad request.
+ *       401:
+ *         description: Authorization information is missing or invalid.
+ *       404:
+ *        description: A user with the specified username was not found.
+ *       5XX:
+ *        description: Unexpected error.
+ */
+router.post('/profiles/:username/follow/:userId', follow);
+
+/**
+ * @swagger
+ *
+ * /api/v1/users:
+ *   post:
+ *     description: Un-follow User
+ *     produces:
+ *       - application/json
+ *     request:
+ *         content:
+ *         - application/json
+ *         schema:
+ *           type: array
+ *           items:
+ *         $ref: '#/definitions/users'
+ *     responses:
+ *       201:
+ *         description: User un-followed
+ *       400:
+ *         description: Bad request.
+ *       401:
+ *         description: Authorization information is missing or invalid.
+ *       404:
+ *        description: A user with the specified username was not found.
+ *       5XX:
+ *        description: Unexpected error.
+ */
+router.delete('/profiles/:username/unfollow/:userId', follow);
```

Comment:
why are you using `/profiles` prefix here?

Reply:
This is how it was stated in the readme